API/DTOs/FilteringRequestDTO.cs

```diff
@@ -59,5 +60,6 @@ public class FilteringRequestDTO
     /// <summary>
     /// A list of osu! player ids that will be filtered
     /// </summary>
-    public required IEnumerable<long> OsuPlayerIds { get; set; } = [];
+    [MinLength(1, ErrorMessage = "At least one osu! player ID is required.")]
```

Comment:
```suggestion
    [MinLength(1)]
```
I have tried to avoid using custom error messages if at all possible